internal/experiments/experiments.go

```diff
@@ -8,14 +8,23 @@ import (
 	"slices"
 	"strings"
 
+	"gopkg.in/yaml.v3"
+
 	"github.com/Ladicle/tabwriter"
 	"github.com/joho/godotenv"
 	"github.com/spf13/pflag"
 
 	"github.com/go-task/task/v3/internal/logger"
 )
 
-const envPrefix = "TASK_X_"
+const (
+	envPrefix             = "TASK_X_"
+	defaultConfigFilename = ".task-experiments.yml"
```

Comment:
Given that we support `yaml` and `yml` for the main Taskfile. We might consider supporting both here too. I think a wider discussion about the file name would be good too. The current name was just a random idea. I'd like to gather some other ideas too and pick the best one(s).